db/migrate/20180926081538_create_instance_user_invitations.rb

```diff
@@ -0,0 +1,18 @@
+class CreateInstanceUserInvitations < ActiveRecord::Migration[5.1]
```

Comment:
Migration doesn't work on latest master, since we removed the schema_plus gems. The following error occurs:

```
ArgumentError: Unknown key: :case_sensitive. Valid keys are: :unique, :order, :name, :where, :length, :internal, :using, :algorithm, :type
/Users/jeremyyap/Documents/coursemology/coursemology2/db/migrate/20180926081538_create_instance_user_invitations.rb:3:in `change'
bin/rails:4:in `require'
bin/rails:4:in `<main>'
```

Remove `index: { case_sensitive: false }`. Instead, you can specify the case-insensitive index like this:

```
    add_index :instance_user_invitations, 'lower((email)::text)'
```

`::text` typecasts the email to a text type which i think can be omitted in this case. The `schema_plus` gems typecast everything by default.